client/src/modules/lead/AddLead/AddLead.js

```diff
@@ -309,11 +319,12 @@ class AddLead extends React.Component {
         domain: this.props.auth.domainid,
         owner: this.props.auth.userid,
         stage: this.state.stage,
-        name: this.state.name.length > 0 ? this.state.name : this.state.namePlaceholder,
+        name: trim(this.state.name).length > 0 ? trim(this.state.name) : trim(this.state.namePlaceholder),
```

Comment:
isBlank does trim and isEmpty so `trim(this.state.name).length > 0` should be `!isBlank(this.state.name)`